### ISSUE TITLE
Grieflogger reload command feature

### ISSUE THREAD

**Issue by creaper3** [DAQEM/GriefLogger#40]
When the mysql database restarts, grieflogger does not reconnect, and spams the console. A reload command or auto-reconnect thing would be awesome.

**Comment by DAQEM** [comment]
Should be fixed.

**Comment by denny4-user** [comment]
how?? where this options ?

**Comment by DAQEM** [comment]
> how?? where this options ?

https://github.com/DAQEM/GriefLogger/blob/1.21.4/common/src/main/java/com/daqem/grieflogger/database/Database.java#L59
`allowReconnect=true&autoReconnect=true`

It does it automatically

**Comment by denny4-user** [comment]
for 1.20.1 ?

**Comment by DAQEM** [comment]
Yes, 1.20.1 does the same thing: https://github.com/DAQEM/GriefLogger/blob/1.20.1/common/src/main/java/com/daqem/grieflogger/database/Database.java#L59

**Comment by denny4-user** [comment]
https://logs.hosting-minecraft.pro/rumixahuho

It doesn't help.. Spamming this error
